Dockerfile.devkit

```
@@ -68,6 +68,7 @@ ARG USER_ID=0
 ARG GROUP_NAME=root
 ARG GROUP_ID=0
 ARG DOCKER_GID=0
+ARG MINDTHEGAP_VERSION=0.19.0
```

Whatever you decide, lets just keep it consistent please https://github.com/mesosphere/dkp-release/pull/25/files#r955354523
```suggestion
ARG MINDTHEGAP_VERSION=v0.19.0
```

---

I don't understand isn't it already consistent with what is in dkp-release?